app/components/log_summary_component.html.erb

```diff
@@ -37,16 +37,18 @@
       <% end %>
 
       <% if current_user.support? %>
-        <% if log.owning_organisation or log.managing_organisation %>
+        <% if log.owning_organisation %>
```

Comment:
Does this not hide managed lettings logs from the summary list?

Reply:
ah yep, I've tweaked this slightly